src/Sarif.Viewer.VisualStudio/ContentTypes.cs

```diff
@@ -3,9 +3,10 @@
 
 namespace Microsoft.Sarif.Viewer
 {
-    using Microsoft.VisualStudio.Utilities;
     using System.ComponentModel.Composition;
```

Comment:
>System [](start = 10, length = 6)

Eddy, I went ahead and merged this. Thanks for getting this in. @v-lgold@microsoft.com, if you want to tinker further with this, go ahead and make the changes you'd like.